ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/local-users.j2

```diff
@@ -1,11 +1,19 @@
 {# eos - local users #}
-{% if local_users is defined %}
+{% if local_users is arista.avd.defined %}
```

Comment:
this check is redundant and should be removed

Reply:
If we remove the first if statement, how can we control the insertion of the ! at the beginning of the block?

Reply:
Good point. So in this case it is ok to have the check.